src/Application/FormMain.cs

```diff
@@ -196,7 +199,7 @@ private void CheckNetwork()
                     client.UseDefaultCredentials = true;
                     try
                     {
-                        string html = client.DownloadString(Utilities.HelpBaseUri);
+                        string html = await client.DownloadStringTaskAsync(Utilities.HelpBaseUri);
```

Comment:
I think the right name is "DownloadStringAsync".  We don't put "Task" in the name of all async methods.

Reply:
There are two overload of this async method in c#. It is not my naming convention but it's microsofts. I just used one of then that returns a task

Reply:
Ah, my bad, missed the fact this was a library call, thanks.